modin/conftest.py

```diff
@@ -452,3 +452,32 @@ def TestReadGlobCSVFixture():
 @pytest.fixture
 def get_generated_doc_urls():
     return lambda: _generated_doc_urls
+
+
+def pytest_sessionstart(session):
+    if TestRayClient.get():
+        import ray
+        import subprocess
+
+        port = "50051"
+        # Clean up any extra processes from previous runs
```

Comment:
I would say it's a bad idea, who knows which processes are running there from some parallel session (potentially by some other user)?

Reply:
Yeah that's reasonably, I'll get rid of those.